### PR TITLE
xcompilation for *bsd in CI

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

**Comment by MarcoPolo** [comment]
How does this affect CI time?

**Comment by heronhaye** [comment]
negligible or probably nothing since its in parallel - try `GOOS=freebsd go build -o kb github.com/keybase/client/go/keybase`

**Comment by heronhaye** [comment]
ping @jzila

File: Jenkinsfile [hunk 1]
```diff
@@ -199,6 +199,20 @@ helpers.rootLinuxNode(env, {
...
                 }
                 parallel (
+                  test_xcompilation: { withEnv([
+                    "PATH=${env.PATH}:${env.GOPATH}/bin",
+                  ]) {
+                    if (hasGoChanges) {
+                      def platforms = ["freebsd", "netbsd", "openbsd"]
+                      for (platform in platforms) {
+                          withEnv(["GOOS=${platform}"]) {
+                              println "Testing compilation on " + platform
```
Comment (jzila) [inline Jenkinsfile]: `println "Testing compilation on ${platform}"` here and below.

**Comment by heronhaye** [comment]
Merging over chat flake

**Pull request merged**